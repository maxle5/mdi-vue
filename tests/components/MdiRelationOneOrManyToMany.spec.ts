
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiRelationOneOrManyToMany from "../../src/components/MdiRelationOneOrManyToMany.vue";

test("MdiRelationOneOrManyToMany snapshot", () => {
  const wrapper = mount(MdiRelationOneOrManyToMany, {});
  expect(wrapper.html()).toMatchSnapshot();
});
