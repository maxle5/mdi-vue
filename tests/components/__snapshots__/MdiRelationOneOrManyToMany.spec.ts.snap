// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiRelationOneOrManyToMany snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M22 13V19H21L19 17H11V9H7V11H5V9L3 11H2V5H3L5 7V5H7V7H13V15H19L21 13Z" style="fill: currentColor;"></path>
</svg>"
`;
